pym/bob/cmds/jenkins.py

```diff
@@ -1797,6 +1808,8 @@ def doJenkinsSetOptions(recipes, argv):
                         help="Undefine environment variable override")
     parser.add_argument("--credentials", help="Credentials UUID for SCM checkouts")
     parser.add_argument('--authtoken', help='AuthToken for remote triggering jobs')
+    parser.add_argument('--shortdescription', action='store_true', default=False,
+                        help='Don\'t calculate all paths for description')
```

Comment:
I think for `set-options` we need the inverse `--longdescription` too. This is like the "keep", "download" and "sandbox" options below. Only one switch can be given at the same time. Without any option the "description" switch is kept unchanged.